bokehjs/src/lib/models/filters/filter.ts

```diff
@@ -37,10 +37,10 @@ export class Filter extends Model {
       if (isArrayOf(filter, isInteger)) {
         return filter
       }
-      logger.warn(`Filter ${this.id}: filter should either be array of only booleans or only integers, defaulting to no filtering`)
+      logger.warn(`${this}: filter should either be array of only booleans or only integers, defaulting to no filtering`)
       return null
     } else {
-      logger.warn(`Filter ${this.id}: filter was not set to be an array, defaulting to no filtering`)
+      logger.warn(`${this}: filter was not set to be an array, defaulting to no filtering`)
```

Comment:
Why is this code `compute_indices` code combined on the base class at all? A filter for a boolean filter should not be a list of integers, these paths seem like they belong on subclasses so that they can be specific